README.md

```diff
@@ -1,3 +1,9 @@
+[![pyang](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyang.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyang.html)
+[![pyang@1.7.8](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyang@1.7.8.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyang@1.7.8.html)
+[![pyangbind](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyangbind.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyangbind.html)
+[![goyang/ygot](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:goyang-ygot.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:goyang-ygot.html)
+[![yanglint](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:yanglint.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:yanglint.html)
+
```

Comment:
Per discussion in the working group meeting, might be nice to include a visual indicator on the status of the OpenConfig linter as well.   Not a blocker, but a nice-to-have IMO.

Reply:
This is [in the works](https://github.com/openconfig/models/pull/855/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8) in the models repo. I will add it to public once it's in models.

Reply:
The badge is working now.